src/features/main_menu/mobile.vue

```diff
@@ -95,14 +95,15 @@
       </v-row>
       <div style="height: 40px" />
 
-      <v-footer color="primary" fixed>
+      <v-footer color="primary" fixed style="padding-bottom: calc(6px + env(safe-area-inset-bottom, 0px))">
         <v-btn x-small dark outlined @click="$refs.optionsModal.show()"
           >Options</v-btn
         >
         <v-spacer />
         <v-btn x-small dark outlined @click="$refs.aboutModal.show()"
           >About</v-btn
         >
+        <v-spacer />
```

Comment:
This is unrelated, but it seemed like an oversight to me that there wasn't horizontal spacing between only these two buttons.